hosts/ghaf-log/configuration.nix

```diff
@@ -46,4 +58,47 @@
       efiInstallAsRemovable = true;
     };
   };
+
+  # Grafana
+  services.grafana = {
+    enable = true;
+
+    settings = {
+      server = {
+        http_port = 3000;
+        http_addr = "127.0.0.1";
+      };
+
+      # disable telemetry
+      analytics = {
+        reporting_enabled = false;
+        feedback_links_enabled = false;
+      };
+
+      # allow read-only access to dashboards without login
+      # this is fine because the page is only accessible with vpn
+      "auth.anonymous".enabled = true;
```

Comment:
`ghaflogs.vedenemo.dev` is world readable and does not need vpn to access unlike our monitoring server where this was copied from. This means anyone can browse our logs.

Reply:
Does it mean `"auth.anonymous".enabled = true;` need to set to false?

Reply:
It's false by default so the line can be removed.

Reply:
Line removed.